tasks/main.yml

```diff
@@ -2,11 +2,24 @@
 # tasks/main.yml: "Master" playbook for the cmacrae.sensu role
 
   - include_vars: "{{ ansible_distribution }}.yml"
+    when: not windows_server
```

Comment:
So, because `windows_server` is undefined, the integration tests are failing in Travis. How are you defining that locally? As a host variable? I assume there's a native Ansible fact that we can key off here instead.

Reply:
I was explicitly defining windows_server = true when invoking the role. I agree; we can check to see if we are on a windows using a built in fact var